lib/siwapp_web/live/invoices_live/edit.html.heex

```diff
@@ -21,7 +21,7 @@
         <%= label(f, :series_id, class: "label") %>
         <p class=" control select is-fullwidth">
           <%= if @live_action == :new do %>
-            <%= select(f, :series_id, Enum.map(@series, &[key: &1.name, value: &1.id, selected: &1.default])) %>
```

Comment:
I don't understand this. Could you explain it?

Reply:
This generates a `select` tag and all its `option` tag children for the formulary `f`.

![image](https://github.com/siwapp/siwapp/assets/144691941/13834c04-8651-4429-829f-f6dca7637fd5)

I have removed `selected:` because we do not want to pick the default value, we want to use the value from the original invoice and also give the chance to pick any valid value.

Reply:
But here the `@live_action` is `:new`. Are we lossing the default functionality?

Reply:
I have had to change the logic a little, but, before, I extracted it from the template and put it into the view.